settings.gradle

```diff
@@ -6,5 +6,6 @@ pluginManagement {
             url = 'https://maven.fabricmc.net/'
         }
         gradlePluginPortal()
+        mavenCentral()
```

Comment:
Is this needed?

Reply:
It appears that it's not necessary. It was necessary 6 hours ago as Gradle kept complaining it could not find the 3.9.0 version of [this artifact](https://bintray.com/bintray/jcenter/org.eclipse.platform%3Aorg.eclipse.equinox.registry/3.9.0). My guess is, Loom is using the [maven central versions](https://mvnrepository.com/artifact/org.eclipse.platform/org.eclipse.equinox.registry/3.9.0) of the packages while the Gradle settings from the example mod are using JCenter to find these packages.

I would say that it's not strictly necessary except when Loom starts to use a new version of a dependency that hasn't been synced to JCenter yet. In my opinion, this would cut down on issues that relate to not finding an artifact that just hasn't synced yet (as I wasn't the only one on FabricMC's Discord today asking about it). After the artifact syncs, the link to Maven Central is no longer needed.

I'd say it's up to you whether or not I should remove the Maven Central link. Maybe we can just leave it commented out if you want.